src/imputation/apportionment.py

```diff
@@ -43,7 +46,7 @@ def update_column(df: pd.DataFrame, col: str) -> pd.Series:
         pd.Series: A single column dataframe with the values in instance 0
         copied to other instances for the same reference.
     """
-    updated_col = df.groupby("reference")[col].transform(sum)
+    updated_col = df.groupby("reference")[col].transform(max)
```

Comment:
Do we need to add a .`order_by` here to make sure it's always instance 0? Just thinking if (for some reason) the instances were in a different order.

Reply:
Doesn't max take the maximum value in the group? If so, we're fine. The point is that only instance 0 has an entry, all other instances are null. I had "sum" there before, but Tom pointed out it wasn't obvious what's going on.

Reply:
Ah this makes sense, if it will always be the case that instance 0 is the only entry!